serial_driver/include/serial_driver/serial_port.hpp

```diff
@@ -0,0 +1,237 @@
+// Copyright 2021 LeoDrive, Copyright 2021 The Autoware Foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef SERIAL_DRIVER__SERIAL_PORT_HPP_
+#define SERIAL_DRIVER__SERIAL_PORT_HPP_
+
+#include <array>
+#include <string>
+
+#include "io_context/common.hpp"
+#include "io_context/io_context.hpp"
+
+using sbp = asio::serial_port_base;
```

Comment:
should this be `spb`?

Reply:
It should be, yes. Thanks. Fixed in d524784.